src/renderers/shared/fiber/__tests__/ReactIncremental-test.js

```diff
@@ -410,7 +410,7 @@ describe('ReactIncremental', function() {
 
     // Init
     ReactNoop.render(<Foo text="foo" text2="foo" step={0} />);
-    ReactNoop.flushLowPri(55);
+    ReactNoop.flushLowPri(55 + 25);
```

Comment:
Because I'm now going through the normal phase I'm now asking `timeRemaining()` more often instead of skipping straight through. That's why these tests now "takes longer" in simulated time.